packages/mockyeah-cli/bin/mockyeah-record.js

```diff
@@ -74,10 +75,10 @@ boot(env => {
           process.exit(1);
         }
 
-        withName(env, answers.name);
+        withName(env, answers.name, { match });
       }
     );
   } else {
-    withName(env, name);
+    withName(env, name, { match });
```

Comment:
:100:  for options object for future extensibility.

Reply:
I know, right?! Super impressed with your foresight. It made plugging in request matching super easy.